src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -98,15 +108,18 @@ public async Task<AuthFlowResult> GetTokenAsync()
                             .GetTokenInteractiveAsync(this.scopes, account, cancellationToken),
                             this.errors)
                             .ConfigureAwait(false);
-                        tokenResult.SetAuthenticationType(AuthType.Interactive);
+                        tokenResult.SetAuthenticationType(AuthType.Interactive); // confirm this.

```

Comment:
@kyle-rader I was testing the code locally and I did not see an interactive prompt at this step. It seemed to have silently authenticate. Am I missing something here? 

I was testing on a machine with windows hello face enabled. I'm assuming it's because of that I couldn't see anything here. Else it would have prompted to take windows hello PIN. Am I right? 

My question here is, the authentication type could be `Interactive` or `Silent` depending on the machine configuration.  So which authentication type can be set here?

Also, I have added a new attribute - `no_of_interactive_prompts` to the schema. Whenever we invoke an interactive prompt, I'm incrementing the `interactivePromptsCount`. Am I right here?